apps/docs/content/guides/database/connection-management.mdx

```diff
@@ -0,0 +1,98 @@
+---
```

Comment:
@TheOtherBrian1 i think we should split this into two, one about Supavisor/Pooler and another one about monitoring connections.

feels a bit weird right now where the first half is all about supavisor and then have grafana/monitoring come in the picture.

Reply:
I considered adding the `monitoring connections` docs into 'connecting-to-postgres'. That would reduce the need for repetition and refreshers between the pages. However, I would like there to be some distance between the two. 

Monitoring connections is for three things:
- debugging connection bottlenecks
- determining optimal compute size
- adjusting pool size when using multiple APIs


These things aren't relevant to those sampling Supabase for the first time. Placing it upfront, creates needless noise and may even make users concerned about management burdens. 

What are your thoughts of shrinking the following passage and just moving it into the serverless drivers docs:
```
You can change how many database connections Supavisor can manage by altering the pool size in the [Database Settings](/dashboard/project/_/settings/database):

![Connection Info and Certificate.](/docs/img/database/pool-size.png)

The general rule is that if you are heavily using the PostgREST database API, you should be conscientious about raising your pool size past 40%. Otherwise, you can commit 80% to the pool. This leaves adequate room for the Authentication server and other utilities.

These numbers are generalizations and assume a certain level of activity from all connected servers. The actual values depend on your peak connection usage. For instance, if you were only using 80 connections in a week period and your database could support 500 connections, then realistically you could allocate the remaining 420 (minus a reasonable buffer) to service more demand.
```

Reply:
let's just keep it in here for now